.github/workflows/_build.yml

```diff
@@ -132,16 +132,6 @@ jobs:
       pull-requests: write
       contents: write
     timeout-minutes: 20
-
-    strategy:
```

Comment:
now that the tests run so much faster it's much more likely that there's a clash between two different variants trying to commit the changes, resulting in a CI failure. so run all variants in one go. the whole thing still takes under 10 minutes anyways